app/keepers/keepers.go

```diff
@@ -351,7 +351,7 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
-	supportedFeatures := "iterator,staking,stargate,osmosis"
+	supportedFeatures := "iterator,staking,stargate,osmosis,cosmwasm_1_1"
```

Comment:
Sanity checking, cosmwasm 1.1 won't break anything existing right

Reply:
> All CosmWasm 1.0 contracts run on 1.0 and 1.1 hosts. All CosmWasm 1.1 contracts run on 1.0 and 1.1 hosts. Smooth, right?

https://medium.com/cosmwasm/cosmwasm-1-1-14233baf8162

Perfect! (though if 1.1 runs on 1.0 hosts, I don't really understand why its feature flagged?)

Reply:
This just makes the feature available so that if someone compiles their contract with 1.1, then the 1.1 features from cosmwasm_std and whatever they feature-gate in their contract will be available. 

The version of wasmd we're on should already support cosmwasm_1.1